### PR TITLE
[0.6->main] Update silkworm to include 0.6.1 fixes

### LINKED ISSUE #83: trace_block api reports wrong mining rewards
Depends on https://github.com/eosnetworkfoundation/silkworm/issues/80.

It should be 0 in our case but this api will return 2 EOS.

This will eventually cause explorer display wrong value.

Real balance seems not affected.

### LINKED ISSUE #95: JSONRPC: eth_call proceed 'input' field unexpectedly for view functions
According to JSON-RPC spec( https://ethereum.github.io/execution-apis/api-documentation/ ), eth_call accepts 'input' as a parameter. 

However, calling eth_call for contract view functions with 'input' filed instead of 'data' filed, the RPC returned 'execution reverted'. Replace 'input' with 'data' works.

Reproduce step for testnet:
non-working:  {"jsonrpc":"2.0","id":1,"method":"eth_call","params":[{"from":"0x0000000000000000000000000000000000000000","input":"0x54fd4d50","to":"0x40d046f41b6e161909054c0f3ae075199d363c23"},"latest"]}
working: {"jsonrpc":"2.0","id":1,"method":"eth_call","params":[{"from":"0x0000000000000000000000000000000000000000","data":"0x54fd4d50","to":"0x40d046f41b6e161909054c0f3ae075199d363c23"},"latest"]}

### PR DESCRIPTION
**Pull request by yarkinwho**
Resolve #95 
Resolve #83 

Should also include test fixes in silkworm but it has noting to do with this repo.

### CODE REVIEW

**Comment by yarkinwho** [comment]
DO NOT MERGE until we verified the commit hash after silkworm finished merge 0.1 to master.

**Comment by yarkinwho** [comment]
Updated to track head of silkworm

**Review by elmato — APPROVED** [review]

**Pull request merged**
